client/tx/query.go

```diff
@@ -116,13 +122,8 @@ func QueryTxRequestHandlerFn(cdc *wire.Codec, cliCtx context.CLIContext) http.Ha
 	return func(w http.ResponseWriter, r *http.Request) {
 		vars := mux.Vars(r)
 		hashHexStr := vars["hash"]
-		trustNode, err := strconv.ParseBool(r.FormValue("trust_node"))
-		// trustNode defaults to true
-		if err != nil {
-			trustNode = true
-		}
 
```

Comment:
I remove this because if rest-server is stared with trust-node, then we can't verify tx proof even if the request requires proof.

Reply:
OK :+1: